src/makeRobotAccountant.js

```diff
@@ -6,7 +6,29 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+  let sum = 0;
+
+  const getSum = (a) => {
+    const result = (b) => {
```

Comment:
`result` is not the best name for a function (but also, you may immediately return it)